cmd/bridge/main.go

```diff
@@ -122,7 +122,8 @@ func main() {
 
 	fLoadTestFactor := fs.Int("load-test-factor", 0, "DEV ONLY. The factor used to multiply k8s API list responses for load testing purposes.")
 
-	fDevCatalogCategories := fs.String("developer-catalog-categories", "", "Allow catalog categories customization. (JSON as string)")
+	fDevCatalogCategories := fs.String("developer-catalog-categories", "", "DEV ONLY. Allow catalog categories customization. (JSON as string)")
+	fUserSettings := fs.String("user-settings", "configmap", "DEV ONLY. Define if the user settings should be stored in a shared ConfigMap or in users browser (configmap | localstorage).")
```

Comment:
We should add validation when reading the flag.

Say what the default is if not set in the description.

Reply:
nit: maybe `user-settings-location`?

Reply:
I'm fine with that. Renamed everything to `userSettingsLocation` or `user-settings-location`